DependencyInjection/HttplugExtension.php

```diff
@@ -203,6 +203,13 @@ private function configurePluginByName($name, Definition $definition, array $con
                 $definition->replaceArgument(1, [
                     'replace' => $config['replace'],
                 ]);
+            case 'base_uri':
+                $baseService = $serviceId.'.base_uri';
+                $this->createUri($container, $baseService, $config['uri']);
```

Comment:
can we s/$baseService/$baseUriService/ and refactor the section above (add_host) to call its variable s/$uriService/$hostUriService/? i would prefer consistent naming and being specific helps here (otherwise we have twice $uriService which is confusing too)